.rubocop.yml

```diff
@@ -9,6 +9,7 @@ AllCops:
     - 'actionpack/lib/action_dispatch/journey/parser.rb'
     - 'railties/test/fixtures/tmp/**/*'
     - 'actionmailbox/test/dummy/**/*'
+    - 'actiontext/test/dummy/**/*'
```

Comment:
I don't think this `put a dummy application inside the framework tests` scale in the rails repository. It does make sense for an external plugin but for things that are inside the rails repository we should be following the same railties tests pattern that we have for all the other frameworks. The dummy application get outdated and we will end up testing those engines against a version behind of the one that is being currently being developed.